modules/carto/src/api/layer-map.ts

```diff
@@ -189,6 +193,20 @@ function domainFromValues(values, scaleType: SCALE_TYPE) {
   return extent(values);
 }
 
+function uniqueSortedByFrequency(values) {
```

Comment:
You can replace `uniqueSortedByFrequency` with `d3.groupSort(values, g => -g.length, d => d)`

Reply:
Thx, I didn't know about this d3 function